src/requests/types/CreditsReceivedResponseDto.ts

```diff
@@ -7,4 +7,8 @@ export class CreditsReceivedResponseDto {
      * Registry's event ID
      */
     public external_id!: string
-}
\ No newline at end of file
+    /**
+      The id of the project developer in Thallo's system. Represents the same as the sender_id in the request
+    */
```

Comment:
```suggestion
    /**
    *  The id of the project developer in Thallo's system. Represents the same as the sender_id in the request
    */
```

Reply:
maybe we should explicitly mention that this is undefined for certain registries and can be implemented upon request.